python/cudf/cudf/_lib/lists.pyx

```diff
@@ -58,3 +61,27 @@ def explode_outer(Table tbl, int explode_column_idx, bool ignore_index=False):
         column_names=tbl._column_names,
         index_names=None if ignore_index else tbl._index_names
     )
+
+
+def sort_lists(Column col, bool ascending, object na_position):
```

Comment:
We could make `na_position` a Python enum to give a more efficient interface here and do the translation from Python string in the higher level Python API.